static/css/loader.css

```diff
@@ -0,0 +1,89 @@
+@import url(http://fonts.googleapis.com/css?family=Oswald);
+
+body{
+  background: #3498db;
+  margin: 150px 0;
+  text-align: center;
+  color: #fff;
+  font-family: 'Oswald', sans-serif;
+}
+.content{
+  letter-spacing: 2px;
+  text-transform: uppercase;
```

Comment:
This has caused all the text in the post to transform to uppercase.
The transform should be limited to the text in the loader only.
Please fix it ASAP.